tests/Fixer/LanguageConstruct/SingleSpaceAfterConstructFixerTest.php

```diff
@@ -2097,6 +2118,13 @@ function bar() {} }',
                 '<?php class Foo { public /* foo */CONST BAR = 9000; }',
                 '<?php class Foo { public  /* foo */CONST BAR = 9000; }',
             ],
+            [
+                '<?php class Foo {
+    public CONST
+        FOO = 9000,
+        BAR = 10000;
+}',
+            ],
```

Comment:
I think a test for a single multi line const is missing, something like
```php
                '<?php class Foo {
    public CONST
        BAR = 10000
   ;
}'
```

Reply:
please add this test as well
```php
<?php
const
    A = 3,
    B = 3
?>
```

Reply:
Thanks!
I added following test case.
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6167/commits/c23255a34491ca4d9c8c77906ef3e62c7cc02aeb#diff-a8e71f915f191772ed88195992ef77c68cf336af23f9cd697e6936b2e18e7326R593-R599

and I had added some unnecessary tests, so I removed them.
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6167/commits/1b204bee5064e20729427dad23f5ce64ffd4667b

Reply:
> I think a test for a single multi line const is missing, something like
> 
> ```
> <?php class Foo {
>     public CONST
>         BAR = 10000;
> }'
> ```

Thanks for the review!
Looking at the test below, it seemed natural to me that a single multiline would be modified to a single line, what do you think?:)
 https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v3.4.0/tests/Fixer/LanguageConstruct/SingleSpaceAfterConstructFixerTest.php#L572-L577
 https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v3.4.0/tests/Fixer/LanguageConstruct/SingleSpaceAfterConstructFixerTest.php#L882-L887